libmscore/lyrics.cpp

```diff
@@ -593,6 +611,23 @@ void Lyrics::undoChangeProperty(Pid id, const QVariant& v, PropertyFlags ps)
             TextBase::undoChangeProperty(id, v, ps);
             return;
             }
+      else if (id == Pid::AUTOPLACE && v.toBool() != autoplace()) {
+            if (v.toBool()) {
+                  // setting autoplace
+                  // reset offset
+                  resetProperty(Pid::OFFSET);
```

Comment:
Should this be `undoResetProperty` instead? Right now this change seems to be not undoable.